app/models/provider.rb

```diff
@@ -32,8 +32,14 @@ class Provider < ApplicationRecord
 
   belongs_to :recruitment_cycle
 
+  # WIP
```

Comment:
I think we should flesh out the comment for additional context or remove this altogether.